tox.ini

```diff
@@ -18,74 +18,70 @@ deps = {[testenv]basedeps}
 setenv =
     pypy: with_gmp=no
     DJANGO_SETTINGS_MODULE=tests.contrib.django_util.settings
-commands = nosetests --ignore-files=test_appengine\.py --ignore-files=test__appengine_ndb\.py {posargs}
+commands = 
+    py.test \
+      --ignore=tests/contrib/test_appengine.py \
+      --ignore=tests/contrib/test__appengine_ndb.py \
+      {posargs}
 
 [coverbase]
 basepython = python2.7
 commands =
-    nosetests \
-      --with-coverage \
-      --cover-package=oauth2client \
-      --cover-package=tests \
-      --cover-erase \
-      --cover-tests \
-      --cover-branches \
-      --ignore-files=test_appengine\.py \
-      --ignore-files=test__appengine_ndb\.py
-    nosetests \
-      --with-coverage \
-      --cover-package=oauth2client.contrib.appengine \
-      --cover-package=oauth2client.contrib._appengine_ndb \
-      --cover-package=tests.contrib.test_appengine \
-      --cover-package=tests.contrib.test__appengine_ndb \
+    py.test \
+      --cov=oauth2client \
+      --cov=tests \
+      --ignore=tests/contrib/test_appengine.py \
+      --ignore=tests/contrib/test__appengine_ndb.py
+    py.test \
+      --cov=oauth2client.contrib.appengine \
+      --cov=oauth2client.contrib._appengine_ndb \
+      --cov=tests.contrib.test__appengine_ndb \
+      --cov=tests.contrib.test_appengine \
+      --cov-append \
       --with-gae \
-      --cover-tests \
-      --cover-branches \
-      --gae-application=tests/data \
-      --gae-lib-root={env:GAE_PYTHONPATH:google_appengine} \
-      --logging-level=INFO \
+      --gae-path={env:GAE_PYTHONPATH:google_appengine} \
+      --gae-project-path=tests/data \
       tests/contrib/test_appengine.py \
       tests/contrib/test__appengine_ndb.py
 deps = {[testenv]deps}
     coverage
-    nosegae
+    pytest-cov
+    # pytest_gae  # TODO: Move to official source when it is updated
```

Comment:
Where does the reference to `GAE_PYTHONPATH` get used?  Should I add a new fixture to update the `PATH` instead?